client-hc/src/main/java/com/graphhopper/api/model/GHGeocodingEntry.java

```diff
@@ -173,6 +189,32 @@ public void setOsmValue(String osmValue) {
         this.osmValue = osmValue;
     }
 
+    @JsonProperty
+    public Double[] getExtent() {
+        if (this.extent == null) {
+            // TODO should we return null instead?
+            return new Double[0];
+        }
+        return this.extent.toGeoJson().toArray(new Double[4]);
+    }
+
+    public BBox getExtendBBox(){
+        return this.extent;
+    }
+
+    @JsonProperty
+    public void setExtent(Double[] extent) {
+        if (extent == null || extent.length == 0) {
+            return;
+        }
+        if (extent.length == 4) {
+            // Extend is in Left, Top, Right, Bottom; which is very uncommon, Photon uses the same
```

Comment:
Has nothing to do with uncommon. There are many different orders - see bbox javadoc

Reply:
I have just seen that this is the "two points" format, but in a GeoJson style, which switches lat, lon. This is indeed similar to how GraphHopper formats the BBox in Geojson. Should I remove the comment?